llvm/include/llvm/IR/IntrinsicsDirectX.td

```diff
@@ -85,6 +85,8 @@ def int_dx_umad : DefaultAttrsIntrinsic<[llvm_anyint_ty], [LLVMMatchType<0>, LLV
 def int_dx_normalize : DefaultAttrsIntrinsic<[LLVMMatchType<0>], [llvm_anyfloat_ty], [IntrNoMem]>;
 def int_dx_rsqrt  : DefaultAttrsIntrinsic<[llvm_anyfloat_ty], [LLVMMatchType<0>], [IntrNoMem]>;
 def int_dx_wave_getlaneindex : DefaultAttrsIntrinsic<[llvm_i32_ty], [], [IntrConvergent, IntrNoMem]>;
+def int_dx_wave_active_sum : DefaultAttrsIntrinsic<[llvm_any_ty], [LLVMMatchType<0>], [IntrConvergent, IntrNoMem]>;
```

Comment:
This could further be split into `int_dx_wave_active_fsum` and `int_dx_wave_active_ssum` to denote the difference in float/signed integer. They were merged into one as the separation is not required for lowering to `WaveActiveOp`. Would appreciate if reviewers weighed in. Could increase readability/documentation.

Reply:
I think it would be good for consistency reasons to rename this to int_dx_wave_active_ssum

Reply:
Is there a difference between float and signed?

Reply:
No difference in how they are lowered to a DXIL op. They will both be lowered to a `WaveActiveOp` with the same flags.